ci-operator/config/netobserv/goflow2-kube-enricher/OWNERS

```diff
@@ -11,11 +11,8 @@ approvers:
 - oliviercazade
 options: {}
 reviewers:
-- andrew-ronaldson
```

Comment:
yep, sorry, I was confused because the `owners` job failed at first, asking me to manually add an OWNER file .. so I wasn't sure anymore if I had to believe this header 😓
I'll fix it, thanks